packages/@lwc/engine-core/src/shared/format.ts

```diff
@@ -38,3 +45,9 @@ export function getErrorComponentStack(vm: VM): string {
 
     return wcStack.reverse().join('\n\t');
 }
+
+export function addErrorComponentStack(vm: VM, error: any): void {
```

Comment:
I don't think it makes sense to have this method in the `format.ts`. Should probably be moved to an `error.ts` shared utility module.